tests/testsuite/build.rs

```diff
@@ -6419,7 +6419,6 @@ fn pipelining_big_graph() {
         .with_stderr_data(
             str![[r#"
 [LOCKING] 61 packages to latest compatible versions
-[COMPILING] b30 v0.5.0 ([ROOT]/foo/b30)
```

Comment:
I havn't known why this need to be removed in MacOS, see https://github.com/rust-lang/cargo/actions/runs/10036500039/job/27734366493


```
thread 'build::pipelining_big_graph' panicked at tests/testsuite/build.rs:6432:10:

---- expected: tests/testsuite/build.rs:6420:13
++++ actual:   stderr
   1    1 | [LOCKING] 61 packages to latest compatible versions
   2      - [COMPILING] b30 v0.5.0 ([ROOT]/foo/b30)
   3    2 | [COMPILING] a30 v0.5.0 ([ROOT]/foo/a30)
   4    3 | [ERROR] don't actually build me
   5    4 | ...
   6    5 | [ERROR] could not compile `a30` (lib) due to 1 previous error
   7    6 | ...
```